src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -483,7 +483,7 @@ class Meta:
                 "queryset": Zaak.objects.all(),
                 "validators": [NotSelfValidator(), HoofdzaakValidator()],
             },
-            "laatste_betaaldatum": {"validators": [UntilNowValidator()]},
+            "laatste_betaaldatum": {"validators": [DateNotInFutureValidator()]},
```

Comment:
The [UntilNowValidator](https://github.com/maykinmedia/commonground-api-common/blob/main/vng_api_common/validators.py#L275) does the same thing, although it does have a different error code, could also override the UntilNowValidator:

```python
class UntilNowValidator(_UntilNowValidator):
    """
    vng_api_common.validators.UntilNowValidator with TIME_LEEWAY
    """

    @property
    def limit_value(self):
        return super().limit_value + timedelta(seconds=settings.TIME_LEEWAY)

```

Reply:
Because `commonground-api-common` had to change, the leeway is now also within UntilNowValidator so it could be changed back.